### PR TITLE
Refreshing partners / reviewers lists

### PR DESCRIPTION
**Pull request by rkrupinski**

### CODE REVIEW

**Comment by rkrupinski** [comment]
@maxim-kht 

`PATCH` `/api/projects/<id>/` expects `invited_partners` in the payload to be a list of partner ids, which is a bit odd and breaks stuff. Could we please make it accept (thus also return) a collection of partners instead (see `GET` `/api/projects/<id>/` for reference)?

cc @marcindo

**Comment by brandinalexander** [comment]
Related to #777

**Comment by rkrupinski** [comment]
@mjaworski This works as expected 👍

File: backend/unpp_api/apps/project/serializers.py [hunk 1]
```diff
@@ -453,42 +453,34 @@ def get_direct_selected_partners(self, obj):
...
+        reviewers = self.initial_data.get('reviewers', [])
+        if reviewers:
+            instance.reviewers.through.objects.exclude(user_id__in=reviewers).delete()
+            instance.reviewers.add(*User.objects.filter(id__in=reviewers))
+        elif 'reviewers' in self.initial_data:
+            instance.reviewers.clear()
 
-        instance.save()
+        focal_points = self.initial_data.get('focal_points', [])
+        if focal_points is not None:
```
Comment (marcindo) [inline backend/unpp_api/apps/project/serializers.py]: Isn't this block removing all focal points if the field 'focal_points' is not present in update request? That's the observed behavior when user updates invited_partners only

Comment (mjaworski) [reply backend/unpp_api/apps/project/serializers.py]: Yeah, I forgot to fix this conditional. thx!

**Pull request merged**